src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -166,10 +166,9 @@ function MoneyRequestView({
     const isInvoice = ReportUtils.isInvoiceReport(moneyRequestReport);
     const isPaidReport = ReportActionsUtils.isPayAction(parentReportAction);
     const taxRates = policy?.taxRates;
-
     const formattedTaxAmount = updatedTransaction?.taxAmount
-        ? CurrencyUtils.convertToDisplayString(updatedTransaction?.taxAmount, transactionCurrency)
-        : CurrencyUtils.convertToDisplayString(transactionTaxAmount, transactionCurrency);
+        ? CurrencyUtils.convertToDisplayString(Math.abs(updatedTransaction?.taxAmount), transactionCurrency)
+        : CurrencyUtils.convertToDisplayString(Math.abs(transactionTaxAmount ?? 0), transactionCurrency);
```

Comment:
Sorry, why this change?

Reply:
The tax amount is negative in some cases and we never want to display it as a negative number